regulations/static/regulations/js/source/views/comment-view.js

```diff
@@ -93,7 +94,15 @@ var CommentView = Backbone.View.extend({
         };
       }).get()
     };
-    window.localStorage.setItem(this.key, JSON.stringify(payload));
+
+    if (payload.comment) {
+      window.localStorage.setItem(this.key, JSON.stringify(payload));
+    }
+    else {
+      // clear comment if input is empty
+      this.clear();
```

Comment:
Hm, you're right. There are users that will submit only attachments. Before trying to approach checking for both empty comment field and no attachments, should we go this way at all? I did this before you put the clear function, but now instead of just spending time discussing edge cases, should I just remove this bit? Since now they have the option of really clearing the content if that's what they intend, anyway.